### PR TITLE
Set connection name and table prefix

### PR DESCRIPTION
**Pull request by cweiske**
Resolves: #8

### CODE REVIEW

File: src/RatkoR/Crate/CrateServiceProvider.php [hunk 1]
```diff
@@ -29,14 +29,17 @@ public function register()
...
-            $db->extend('crate', function($config)
+            $db->extend('crate', function($config, $name)
             {
+                $config['name'] = $name;
+
                 $connector = new Connector();
                 $connection = $connector->connect($config);
 
                 $database = $config['database'] ?: 'doc';
+                $prefix   = $config['prefix']   ?: '';
```
Comment (RatkoR) [inline src/RatkoR/Crate/CrateServiceProvider.php]: Tests fail if user does not have prefix set in config. Can you update PR and check if prefix key actually exists. Something like:

```
$prefix   = isset($config['prefix']) ? $config['prefix'] : '';
```

**Review by RatkoR — CHANGES_REQUESTED** [review]
See my comment for $prefix variable. Can you update the PR? I'll do it if you're out of time.

File: src/RatkoR/Crate/CrateServiceProvider.php (hunk 1 above)
Comment (cweiske) [reply src/RatkoR/Crate/CrateServiceProvider.php]: will do.

**Comment by cweiske** [comment]
fixed.

**Review by RatkoR — APPROVED** [review]

**Pull request merged**